schema/spacewalk/common/data/namespace.sql

```diff
@@ -687,6 +687,9 @@ INSERT INTO access.namespace (namespace, access_mode, description)
 INSERT INTO access.namespace (namespace, access_mode, description)
     VALUES ('systems.migration', 'W', 'Perform product migration on systems')
     ON CONFLICT (namespace, access_mode) DO NOTHING;
+INSERT INTO access.namespace (namespace, access_mode, description)
+    VALUES ('systems.appstreams', 'W', 'Manage AppStreams on systems (SSM)')
+    ON CONFLICT (namespace, access_mode) DO NOTHING;
```

Comment:
If we're going to use the existing `systems.software.appstreams` namespace, we don't need to insert anything to the `namespace` table.
```suggestion
```